styles/tokens/masthead.json

```diff
@@ -387,7 +387,7 @@
           },
           "color": {
             "comment": "The color of the Selector Dropdown title, found in a Masthead with Extended Navigation.",
-            "value": "{black-tint.60.value}",
+            "value": "{black.value}",
```

Comment:
what motivated this color change?

Reply:
The grey was failing contrast checks. Jane said I talked about it and landed on making it black instead.